pages/about/index.tsx

```diff
@@ -239,6 +239,18 @@ const About: NextPage<{ teams: TeamSection[] }> = ({ teams }) => {
         <H2>{t('about_page.media_mentions_title')}</H2>
         <ArticlesList articles={articles} />
 
+        <H2>{t('about_page.support_us_title')}</H2>
+        <Trans
+          i18nKey={'about_page.support_us_description'}
+          components={[
+            <TextLink href={'https://drive.google.com/drive/folders/1milRfoG2fPsod7moVKeCPM9ikm00kXup'} target="_blank" key="plakatky" />,
+            <TextLink href={'https://www.movapp.cz/contacts'} target="_blank" key="kontakt" />,
```

Comment:
Ten odkaz na kontakt nemůže být absolutní "https://www.movapp.cz/contacts", potřebujeme, aby na polské verzi vedl na polskou stránku s kontaktem, na ukrajinské na ukrajinskou atd... je jednoduchá cesta, jak to zapsat

Reply:
chápu, opravím